http_request_translator/base.py

```diff
@@ -186,3 +193,22 @@ def encode_url(self, url):
         if self.details.get('data') and (self.details.get('method', '').lower() in http_verb_with_encoding):
             encoded_url += quote(self.details['data'], '')
         return encoded_url
+
+    @staticmethod
+    def load_attributes(cls):
+        """Loads attributes to Script class from a given script's template
+
+        Imports the template file/module, assigns all the attributes defined in the template file to the given class.
+
+        :param class cls: Script class to which template is to be loaded.
+        :param str template: Name of the corresponding template file kept in the default template directory.
+        """
+        if not hasattr(cls, '__language__'):
+            raise ValueError("__language__ not found in class: {}, attributes cannot be loaded".format(cls.__name__))
+
+        template = import_module("{templates_path}.{class_template}".format(templates_path=cls._templates_path,
```

Comment:
Please use the coding style:
```python
something(
    foo,
    bar)
```

Aligning the new line that is not necessarily a multiple of four by hitting the spacebar 3 times for instance is quickly annoying.